### PR TITLE
add more test cases

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

File: efficientdet/keras/inspector_test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if os.path.exists(self.savedmodel_dir):
+      shutil.rmtree(self.savedmodel_dir)
+
+  def tearDown(self):
+    super().tearDown()
+    shutil.rmtree(self.tempdir)
+
+  def test_dry(self):
+    result = subprocess.run(
+        ['python3', 'keras/inspector.py', '--mode=dry', '--ckpt_path=_'])
```
Comment (mingxingtan) [inline efficientdet/keras/inspector_test.py:44]: Maybe we we could make it simpler by setting flags? such as:

from keras import inspector
from absl import flags
FLAGS = flags.FLAGS


def test_dry(self):
  FLAGS.mode = 'dry'
  FLAGS.ckpt_path = '_'
  inspector.main(None)
  self.assert(...) 

What do you think?

**Review by mingxingtan — COMMENTED** [review]
Thanks for adding these tests.

File: efficientdet/keras/inspector_test.py (hunk 1 above)
Comment (fsx950223) [reply efficientdet/keras/inspector_test.py:44]: FLAGS shares values in different test cases which breaks other test cases.
For example: ```infer_test``` gets saved_model_dir which is saved in ```export_test```.

Comment (mingxingtan) [reply efficientdet/keras/inspector_test.py]: you can use flagsaver. I will merge this PR, and send you a new PR to show the usage.

File: efficientdet/keras/inference_test.py [hunk 2]
```diff
@@ -24,20 +24,62 @@ class InferenceTest(tf.test.TestCase):
...
+      self.assertEqual(tf.reduce_mean(scores), 0.01000005)
+      self.assertEqual(tf.reduce_mean(classes), 1)
+      self.assertEqual(tf.reduce_mean(valid_lens), 100)
+    elif policy.name == 'float16':
+      pass
+    elif policy.name == 'bfloat16':
+      pass
     self.assertEqual(boxes.shape, (1, 100, 4))
     self.assertEqual(scores.shape, (1, 100))
     self.assertEqual(classes.shape, (1, 100))
```
Comment (mingxingtan) [inline efficientdet/keras/inference_test.py:85]: This file doesn't have the tf.test.main. I wonder how did you run this test?

Comment (fsx950223) [reply efficientdet/keras/inference_test.py:85]: By Pycharm.

**Pull request merged**